source/shared/net/resources/glossary.rst

```diff
@@ -197,6 +211,13 @@ Decryption key
 Dual to :ref:`encryption key<glossary-encryption-key>`. In contrast to the encryption key, which is public,
 this key is only known to the account holder.
 
+.. _glossary-delegate:
+
+Delegate
+========
+
+The action of giving a baker the right to stake an amount of CCD from you in order to increase probability of baking a block and earning rewards. Delegators earn rewards, minus a commission to the baker, on delegated stake.
```

Comment:
Maybe something like (for delegator):

An account that contributes stake to a baker pool, or to passive delegation.
When an account becomes a delegator, the delegated amount of CCD is locked so that it cannot be spent or transferred while it is delegated.
Delegators earn rewards, minus a commission to the baker, in proportion to their delegated stake.